charts/hub/acs/central/templates/job-create-auth-provider.yaml

```diff
@@ -27,7 +27,9 @@ spec:
               #!/usr/bin/env bash
               echo "Creating auth provider in RHACS"
               export DATA={\"name\":\"local-cluster\"}
-              curl -X POST -u "admin:$PASSWORD" -k https://central/v1/authProviders --data '{"name": "OpenShift OAuth", "type": "openshift", "enabled": true}'
+              curl -X POST -u "admin:$PASSWORD" -k https://central/v1/authProviders --data '{"name": "OpenShift OAuth", "type": "openshift", "enabled": true}' > /tmp/output.json
+              AUTH_PROVIDER_ID=$(sed 's/,/\n/g' /tmp/output.json | grep -w id | awk -F\" '{ print $4 }') ### Since the image does not have the 'jq' command
+              echo "curl -X POST -u "admin:$PASSWORD" -k https://central/v1/groups -s --data '{\"roleName\":\"Admin\",\"props\":{\"authProviderId\":\"$AUTH_PROVIDER_ID\",\"key\":\"name\",\"value\":\"kube:admin\"}}' ### Adding the Admin entry for the KubeAdmin user" | bash
```

Comment:
should it just echo the command or perform it??

Reply:
wait, we're piping it through bash??

Reply:
Unfortunately yes, because of the strange way bash variables are expanded as there are multiple curly brackets involved in the curl command. The only way that I can think of is this one as I wrote in the previous comment: echo the command so the variables are correctly expanded and then pipe the output through bash so that everything is in place.

Reply:
I didn't thought of that :) trying right now!

Reply:
Thanks for the hint, Andrew! I have just edited the files and it looks definitely better now :D